frame/support/procedural/src/construct_runtime/mod.rs

```diff
@@ -299,14 +325,10 @@ fn decl_outer_event_or_origin<'a>(
 			None => {}
 		}
 	}
-	let macro_call = match kind {
-		DeclOuterKind::Event => quote!(#scrate::impl_outer_event!),
-		DeclOuterKind::Origin => quote!(#scrate::impl_outer_origin!),
-	};
-	let enum_name = Ident::new(kind_str.as_str(), Span::call_site());
+
 	Ok(quote!(
-		#macro_call {
-			pub enum #enum_name for #runtime_name where system = #system_name {
+		#scrate::impl_outer_event! {
+			pub enum Event for #runtime_name {
```

Comment:
Why is `Event` now a hard-coded name?

Reply:
i think this was @bkchr 's change...

Reply:
It was always named `Event` in `construct_runtime!`. Before this change, the method could be used either for `Origin` or for `Event`. Now these are split and I hard coded the name.